### PR TITLE
fixing install scripts

### PR DESCRIPTION
**Pull request by doktoric**
@lalyos

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -3,16 +3,11 @@
...
-        -w "%{redirect_url}" \
-        https://github.com/sequenceiq/cloudbreak-deployer/releases/latest
-    )
-    local latest=${redirect##*v}
+    mkdir -p cloudbreak-deployer
+    cd cloudbreak-deployer
     local osarch=$(uname -sm|tr " " _)
 
-    local url="https://github.com/${GIT_ORG}/${GIT_PROJECT}/releases/download/v${latest}/${GIT_PROJECT}_${latest}_${osarch}.tgz"
+    local url="https://github.com/${GIT_ORG}/${GIT_PROJECT}/releases/download/v1.1.0/${GIT_PROJECT}_1.1.0_${osarch}.tgz"
```
Comment (lalyos) [inline install]: Please introduce a `CBD_VERSION` variable to the top of the script, so we can automate the replacement

File: install [hunk 2]
```diff
@@ -3,16 +3,11 @@
...
 main() {
-    local redirect=$(
-    curl -s \
-        -o /dev/null \
-        -w "%{redirect_url}" \
-        https://github.com/sequenceiq/cloudbreak-deployer/releases/latest
-    )
-    local latest=${redirect##*v}
+    mkdir -p cloudbreak-deployer
+    cd cloudbreak-deployer
```
Comment (lalyos) [inline install]: `cd $_`

**Comment by lalyos** [comment]
please add some changelog/docs (even 1 line)

**Comment by doktoric** [comment]
done

**Pull request merged**